tests/ShopBundle/Functional/Model/Order/OrderFacadeTest.php

```diff
@@ -71,7 +68,7 @@ public function testCreate()
         $orderData->street = 'street';
         $orderData->city = 'city';
         $orderData->postcode = 'postcode';
-        $orderData->country = $persistentReferenceFacade->getReference(CountryDataFixture::COUNTRY_CZECH_REPUBLIC_1);
+        $orderData->country = $persistentReferenceFacade->getReference(CountryDataFixture::COUNTRY_CZECH_REPUBLIC);
```

Comment:
is this part of cart refactoring commit?

Reply:
I dont think so, but it is just not mentioned.

Reply:
it is part of the country unification PR, i will add it into upgrade.md PR